homeassistant/components/octoprint/strings.json

```diff
@@ -21,7 +21,8 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "unknown": "[%key:common::config_flow::error::unknown%]",
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
-      "auth_failed": "Failed to retrieve application api key"
+      "auth_failed": "Failed to retrieve application api key",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
```

Comment:
We're missing strings for the reauth confirm step flow.

Reply:
Added